src/frontend/Komponenter/Behandling/Simulering/SimuleringTabellWrapper.tsx

```diff
@@ -79,6 +82,13 @@ const SimuleringTabellWrapper: React.FC<{
         return simuleringsresultat.feilutbetaling > 0;
     }
 
+    function erUnder4xRettsgebyr() {
+        return (
+            simuleringsresultat.feilutbetaling < 5108 &&
```

Comment:
Magic-number - bør dette tallet komme fra et sted? Eller være en konstant som er mulig å finne igjen?

Vi får jo en "runddans" når vi kommer til jul og skal oppdatere dette tallet i systemene våre 🤔 Blir fort `ef-sak`, `ef-sak-frontend`, `tilbake` og `tilbake-frontend` ?

Reply:
Sant. Kanskje kontrakter? For frontend-apper så hjelper kanskje ikke det så mye. 
Lage et endepunkt for det i familie-tilbake?

Reply:
Usikker på "slutteffekten" - det kan vi kanskje diskutere på kontoret en dag?
I første omgang ville jeg hatt noe ala `const RETTSGEBYR_BELØP = 1277; const fireRettsgebyr = 4*RETTSGEBYR_BELØP` eller lignende for å kunne finne det igjen senere